Demo/EarlGreyExample/EarlGreyExample.xcodeproj/project.pbxproj

```diff
@@ -562,6 +561,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OBJC_BRIDGING_HEADER = "";
 				SWIFT_OBJC_INTERFACE_HEADER_NAME = "EarlGreyExampleSwift-Swift.h";
+				SWIFT_VERSION = "";
```

Comment:
i think you should remove the default value from the project level. unspecified can also indicate it has swift sources, right?